base_layer/core/src/validation/block_validators.rs

```diff
@@ -0,0 +1,146 @@
+// Copyright 2019. The Tari Project
+//
+// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+// following conditions are met:
+//
+// 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following
+// disclaimer.
+//
+// 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
+// following disclaimer in the documentation and/or other materials provided with the distribution.
+//
+// 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote
+// products derived from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+// INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+// DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+// WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+// USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use crate::{
+    blocks::{Block, BlockValidationError},
+    chain_storage::{BlockchainBackend, BlockchainDatabase},
+    validation::{Validation, ValidationError},
+};
+use std::sync::Arc;
+use tari_transactions::{consensus::ConsensusRules, types::CryptoFactories};
+
+/// This validator tests whether a candidate block is internally consistent
+pub struct StatelessValidator {
+    rules: Arc<ConsensusRules>,
+    factories: Arc<CryptoFactories>,
+}
+
+impl StatelessValidator {
+    pub fn new(rules: Arc<ConsensusRules>, factories: Arc<CryptoFactories>) -> Self {
+        Self { rules, factories }
+    }
+}
+
+impl<B: BlockchainBackend> Validation<Block, B> for StatelessValidator {
+    /// The consensus checks that are done (in order of cheapest to verify to most expensive):
+    /// 1. Is there precisely one Coinbase output and is it correctly defined?
+    /// 1. Is the accounting correct?
```

Comment:
```suggestion
    /// 2. Is the accounting correct?
```

Reply:
This is correct markdown (see comment source)

1. "1."
1. means
1. a
1. numbered 
1. list 
1. in
1. markdown

Reply:
I see, that is Cool.